src/tinypeel/tinypeel.py

```diff
@@ -277,12 +277,20 @@ def getArgs():
         help="Flag to enable writing out the genotype probabilities.",
     )
     output_parser.add_argument(
-        "-calling_threshold",
+        "-geno_call_threshold",
         default=None,
         required=False,
         type=float,
         nargs="*",
-        help="Genotype calling threshold(s). Multiple space separated values allowed. Use. .3 for best guess genotype.",
+        help="Genotype calling threshold(s). Multiple space separated values allowed. Any value less than 1 / 3 would be replaced by 1 / 3.",
```

Comment:
@XingerTang add the "Any value less than 1 / 3 would be replaced by 1 / 3."  to `usage.rst` too.